#goal definition
geometry_msgs/Pose Pose_B
---
#result definition
geometry_msgs/Point[] obstacles
---
#feedback
Header head
string status